contributed_definitions/nyaml/NXapm_hit_finding.yaml

```diff
@@ -69,12 +83,164 @@ NXapm_hit_finding(NXprocess):  # when evolving these ideas further inherit from
       Multiplicity must not be confused with how many atoms of the same element
       a molecular ion contains (which is instead encoded with the
       isotope_vector field of each :ref:`NXion` instance).
-    unit: NX_UNITLESS
-    dim: (p,)
-    # the following two quantities are relicts from ePOS files used to give some
-    # insight into the results of the hit_finding algorithm of IVAS/APSuite but typically
-    # used only in the context to learn about the multiplicity of an ion.
-    # pulses_since_last_ion(NX_UINT):
-    #   dim: (n,)
-    # pulse_identifier(NX_INT):
-    #   dim: (n,)
+    dimensions:
+      rank: 1
+      dim: [[1, p]]
```

Comment:
other changes in dim were done towards the new convention, here we go backwards.